requirements/specifications/media/media-info.md

```diff
@@ -61,263 +58,338 @@ I want to show an audio/videophile overlay with detailed information:
+
+If no display is present, a value of zero is returned for both the width and height.
+
+### 5.4. Display Resolution
+
+The `Display` module **MUST** have a `resolution` method that returns the width and height of the display's native resolution.
+
+If no display is present, both returned values **MUST** be zero.
 
-The `hdr` method **MUST** have an optional `resolution` parameter of type `Media.Resolution`.
+### 5.5. Display Resolution Name
 
-If the `resolution` parameter is not provided the method should return values based on the current resolution.
+The `Display` module **MUST** have a `resolutionName` method that returns a user-friendly name of the display's resolution as defined by `Media.ResolutionName`.
 
-### 5.2. Display Width and Height
+If no display is present, a value of `unknown` is returned.
 
-The `Display` module **MUST** have a `width` and `height` method that return the width and height of the display, in centimeters from the HDMI edid.
+### 5.5. Refresh Rate
 
-For built-in displays `width` and `height` **MUST** also be provided.
+The `Display` module **MUST** have a `refreshRate` method that returns an number value for the refresh rate that the display supports (in Hz).
 
-### 5.3. Current and Optimal Resolution
+If no display is present, a value of zero is returned.
 
-The `Display` module **MUST** have a `currentResolution` and  `optimalResolution` method that returns a valid resolution for either the current state of the display, or the displays optimal resolution value.
+### 6. Device Properties
 
-This method **MUST** return a value from the `Display.Resolution` enum.
+Apps need to know various aspects of the device, including its media playing capabilities.
 
-The `optimalResolution` **MUST** come from the HDMI edid.
+These will be surfaced in the `Device` module.
 
-For built-in displays `optimalResolution` **MUST** also be provided.
+Access to these APIs is governed by the `xrn:firebolt:capability:device:info` capability.
 
-### 5.4. Supported Resolutions
- 
-The `Display` module **MUST** have a `supportedResolutions` method that returns an array of valid resolutions that the display supports.
+### 6.1. Current Video Mode
 
-This method **MUST** return a value from the `Display.Resolution` enum.
+The `Device` module **MUST** have a `videoMode` method that returns the current video modes used by the device.
 
-### 5.5. Device Supported Resolutions
-The `Device` module **MUST** have a `supportedResolutions` method that returns an array of valid resolutions that the device supports, regardless of any connected display.
+This method **MUST** return a value from the `Media.VideoMode` enum.
 
-This method **MUST** return an array with one or more of the following values from the `Display.Resolution` enum.
+If no display is present, a value of `unknown` is returned.
 
-### 5.6. Use Source Framerate
-The `Display` module **MUST** have a boolean `useSourceFrameRate` API.
+### 6.1. Video Modes Supported
```

Comment:
same section number twice in a row. there's a great set of plugins for VS Code that auto-numbers your sections. I'll send you the link.